composer.json

```diff
@@ -31,12 +31,16 @@
         "symfony/console": "^3.4",
         "symfony/css-selector": "^3.4",
         "symfony/form": "^3.4",
+        "symfony/framework-bundle": "^3.4",
         "symfony/intl": "^3.4",
         "symfony/lts": "^3.0",
+        "symfony/monolog-bundle": "^3.1",
         "symfony/security": "^3.4",
         "symfony/security-csrf": "^3.4",
+        "symfony/swiftmailer-bundle": "^2.6",
```

Comment:
This is going to block Symfony 4 as well. But we need to upgrade to Swiftmailer 6 to use a newer Swiftmailer bundle.